presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/compose/InstructionStepUi.kt

```diff
@@ -25,39 +25,41 @@ internal fun InstructionStepUi(
     iconTintColor: Color?,
     title: String?,
     detail: String?,
+    nextButtonText: String,
```

Comment:
FYI: Some of Lynn's designs for instruction steps have NULL for the forward button text